### DISCUSSION TITLE
no .sav file in the ROM directory

### DISCUSSION THREAD

**Discussion by BOCHITHEBROCK** [visualboyadvance-m/visualboyadvance-m#1189] [General]
### Expected behavior?

.sav file appears when i save the game

### Actual behavior?

![image](https://github.com/visualboyadvance-m/visualboyadvance-m/assets/147720402/fb95f4a8-1859-426e-bca5-1ba69513ed09)
.sav files wont appear when i save, my settings is on Flash 128k, thats mostly it. I see my .sgm files but other than that it's just .sgm and no .sav

### Steps to reproduce the problem

1.Open VBA-M
2.Open Pokemon Emerald
3.Go in the game, and save within it
4.Check files (Mine is my directory folder for my .sgm and .sav files)
5.
...

### Error messages?

If this is a crash bug, and you have the time and inclination, please provide a symbolic backtrace.
Instructions for doing so are [here](https://github.com/visualboyadvance-m/visualboyadvance-m/blob/master/README.md#reporting-crash-bugs).

### What game were you playing

<Some of this information can be provided with a screenshot of Rom Information, SAVE TYPE for GBA titles can be obtained from Options > Game Boy Advance > Config>
Title: Pokemon Emerald
GB, GBC or GBA: GBA
ROM CODE: Where do I find the ROM CODE?
BIOS Used?: (Share crc hash)
SAVE TYPE: (Attach Battery or Savestate here) I cant attach my .sgm file for some reason..
Clean, Intro'd or Hacked?:(Attach Patch here)

### What settings are you using?

Video driver (simple, opengl):
Sound driver (directsound, xaudio, openal, SDL):
Scale Filter Used?:

### Specifications and Build details

VBA-M: (include commit version number and whether you are using the 32-bit or 64-bit version)
OS: (include version and OS build number using WinVer if on Windows):
If using Linux, specify if using xorg or Wayland:
CPU:
GPU:
RAM:

### Copyright Compliance

- [X] There are no infringing files attached to this bug report.

**Comment by Squall-Leonhart** [comment]
check your directory settings, they aren't necesarily going into the same location.
